scripts/buildImage.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+cd "$(dirname -- "$0")"
+echo "Changed directory to $(pwd)"
+
+echo "Building image..."
+docker build -t ui5-tooling/mkdocs-material -f Dockerfile .
```

Comment:
Regarding the caching:
- locally: yes, this would make it more convenient by tagging it with a version
- For GH Actions: It'd always be built. We do not cache it there. I'm not sure whether we should as in terms of time, it takes some negligible seconds.

Regarding the docker image existence check, I was having some thoughts in the same direction, but can't explain why didn't do it earlier 😆 

On the last point, regarding the tagging with the sample code, I'm not sure I fully grasp the idea 🤔 
`ui5-tooling/mkdocs-material` would always be `latest` and would always be (re-)build.
The `ui5-tooling/mkdocs-material:$DOCKER_TAG` would derrive from `ui5-tooling/mkdocs-material` and would be versioned with git's latest hash

I don't quite understand why we need both `ui5-tooling/mkdocs-material:$DOCKER_TAG` & `ui5-tooling/mkdocs-material`.

Which of those images/tags would we use in `serveDocs.sh`, `buildDocs.sh`, and eventually on the CI?

Reply:
We've agreed to try to find a cross-platform independent way to check whether the Docker file has been modified. That way we'd be able to determine when we need to rebuild the Docker image and when not. This scenario is quite valid for local development. 
The solution with git hash might not be that convenient as the developer must first commit before testing/executing the image building